.gitignore

```diff
@@ -27,7 +27,6 @@ keys
 
 cmd/neutron_query_relayer/neutron_query_relayer
 storage
-neutron_query_relayer
```

Comment:
why?

Reply:
ah, he moved build artifact to build/ folder